### ISSUE TITLE
Sequel Ace could not initialize ssh connection worker failed to initialize ssh handle

### ISSUE THREAD

**Issue by dhildreth-pathlabs** [Sequel-Ace/Sequel-Ace#1690]
- Sequel Ace Version (including build number): 4.0.5 Build 20046
- Sequel Ace Source (App Store/GitHub/Homebrew): Homebrew
- macOS Version: Monterey 12.6
- Processor Type (Intel/Apple): Apple
- MySQL Version: 8.0.28
- macOS Localization: US-EN

**Is Issue Present in [Latest Beta](https://github.com/Sequel-Ace/Sequel-Ace/releases)?**
Not tested

**Description**
When attempting to connect MySQL via SSH, this error occurs:
 ```Sequel Ace could not initialize ssh connection worker failed to initialize ssh handle```
 
 When running SSH command manually from the command line (ie. `ssh user@host`) it connects successfully.
 
 I have verified that the SSH key, SSH config, and SSH known_hosts are all listed in Accessible Files.  I have verified all settings in  Network as well.

**Steps To Reproduce**
1. Create new connection
2. Select SSH as connection type
3. Specify MySQL connection information
4. Specify SSH tunnel connection information with SSH key
5. Click "Test Connection" or "Connect"

**Expected Behaviour**
A connection via SSH is attempted and successful

**Related Issues**
I could not find any with this exact same error message

**Additional Context**
Please let me know if there are any logs I can view.  I'm not familiar with debugging Sequel Ace.

**Comment by Jason-Morcos** [comment]
Can you share an appropriately obfuscated copy of your ash config and a screenshot of your connection by setup?

**Comment by dhildreth-pathlabs** [comment]
I'm in a bit of a strange position now.  I actually wrote this up because a coworker was getting this issue after I suggested Sequel Ace.  I'll ask him if he'd like to contribute to this ticket with the information you requested, but I'm not holding him to it.  He was needing to get some work done, so he downloaded another query browser which worked on the first try (same SSH setup).

If it's any help, we tediously compared all versions and preferences against my working setup and his broken setup.  The only significant difference we found was he has Apple silicon and I have Intel.
